lib/utils/__tests__/isStandardSyntaxDeclaration.test.js

```diff
@@ -1,241 +1,179 @@
 	});
 
 	it('scss list', () => {
-		scssDecls('$list: (key: value, key2: value2)', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(scssDecl('$list: (key: value, key2: value2)'))).toBeFalsy();
 	});
 
 	it('sass map', () => {
-		sassDecls('$map: (value, value2)', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(sassDecl('$map: (value, value2)'))).toBeFalsy();
 	});
 
 	it('scss map', () => {
-		scssDecls('$map: (value, value2)', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(scssDecl('$map: (value, value2)'))).toBeFalsy();
 	});
 
 	it('nested sass var', () => {
-		sassDecls('a\n\t$var: b', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(sassDecl('a\n\t$var: b'))).toBeFalsy();
 	});
 
 	it('nested scss var', () => {
-		scssDecls('a { $var: b }', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(scssDecl('a { $var: b }'))).toBeFalsy();
 	});
 
 	it('nested sass list', () => {
-		sassDecls('a\n\t$list: (key: value, key2: value2)', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(
+			isStandardSyntaxDeclaration(sassDecl('a\n\t$list: (key: value, key2: value2)')),
+		).toBeFalsy();
 	});
 
 	it('nested scss list', () => {
-		scssDecls('a { $list: (key: value, key2: value2) }', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(
+			isStandardSyntaxDeclaration(scssDecl('a { $list: (key: value, key2: value2) }')),
+		).toBeFalsy();
 	});
 
 	it('sass nested property', () => {
-		sassDecls('a\n\tborder:\n\t\tstyle: solid\n\t\tcolor: red', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(sassDecl('a\n\tborder:\n\t\tstyle: solid'))).toBeFalsy();
 	});
 
 	it('scss nested property', () => {
-		scssDecls('a { border: { style: solid; color: red; } }', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(scssDecl('a { border: { style: solid; } }'))).toBeFalsy();
 	});
 
 	it('nested sass map', () => {
-		sassDecls('a\n\t$map: (value, value2)', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(sassDecl('a\n\t$map: (value, value2)'))).toBeFalsy();
 	});
 
 	it('nested scss map', () => {
-		scssDecls('a { $map: (value, value2) }', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
-	});
-
-	it('less var', () => {
-		lessDecls('@var: b', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
-	});
-
-	it('nested less var', () => {
```

Comment:
[note] The following test cases are removed. See the PR description.

- `'less var'`
- `'nested less var'`